service/controller/v24/adapter/adapter.go

```diff
@@ -38,14 +38,14 @@ type Config struct {
 	ControlPlanePeerRoleARN         string
 	ControlPlaneVPCCidr             string
 	CustomObject                    v1alpha1.AWSConfig
-	Clients                         Clients
```

Comment:
This PR now also drops all network dependencies from the adapter. The adapter magic will move to the TCCP resource in the next steps.